### PR TITLE
Adding fsharp support.

### PR DESCRIPTION
**Pull request by navteniev**
The change to add fsharp support turned out to be very simple.  The fsharp project files are the same as csharp project files.

### CODE REVIEW

**Comment by navteniev** [comment]
@ALANVF had a chance to add F# support.  Take a look.

File: internal/backends/dotnet/dotnet.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ var DotNetBackend = api.LanguageBackend{
 	Name:             "dotnet",
 	Specfile:         findSpecFile(),
 	Lockfile:         lockFileName,
-	FilenamePatterns: []string{"*.cs", "*.csproj"}, // C# support so far
+	FilenamePatterns: []string{"*.cs", "*.csproj", "*.fs", "*.fsproj"}, // C# support so far
```
Comment (ALANVF) [inline internal/backends/dotnet/dotnet.go:14]: You might want to remove the comment there

**Review by ALANVF — CHANGES_REQUESTED** [review]
Would be nice if there tests for this. Otherwise looks fine :+1:

File: internal/backends/dotnet/dotnet.go (hunk 1 above)
Comment (navteniev) [reply internal/backends/dotnet/dotnet.go]: Removed the comment

**Comment by navteniev** [comment]
@ALANVF checkout the tests for backends it checks the logic for looking up the backend based on the files in the current directory.

**Review by ALANVF — APPROVED** [review]
Looks good! (except for a minor typo but eh)

**Pull request merged**